.github/workflows/publish-packages.yml

```
@@ -26,11 +29,11 @@ jobs:
       fail-fast: false
       matrix:
         build:
-          - { NAME: linux-x64-glibc,  OS: ubuntu-latest,  TOOLCHAIN: stable, TARGET: x86_64-unknown-linux-gnu }
-          - { NAME: linux-arm64-glibc, OS: arm64,         TOOLCHAIN: stable, TARGET: aarch64-unknown-linux-gnu }
-#          - { NAME: win32-x64-gnu,     OS: windows-latest, TOOLCHAIN: stable, TARGET: x86_64-pc-windows-gnu }
-          - { NAME: darwin-x64,        OS: macos-latest,  TOOLCHAIN: stable, TARGET: x86_64-apple-darwin }
-          - { NAME: darwin-arm64,      OS: macos-14,      TOOLCHAIN: stable, TARGET: aarch64-apple-darwin }
+          - { NAME: linux-x64-glibc,  OS: ubuntu-latest,  TOOLCHAIN: 1.93.0, TARGET: x86_64-unknown-linux-gnu }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)